RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -88,6 +88,7 @@
 ## The muon bad track filter (2016) ________________________________________________||
 from RecoMET.METFilters.BadPFMuonFilter_cfi import *
 
+from RecoMET.METFilters.hfNoisyHitsFilter_cfi import *
```

Comment:
either this, to be consistent with the master version
```suggestion
#HF noise filter
from RecoMET.METFilters.hfNoisyHitsFilter_cfi import *
```
or remove both the import and ` #  hfNoisyHitsFilter *` in the `metFilters` below, since as discussed in https://github.com/cms-sw/cmssw/pull/31413/files#r486415431 an update in this file is not really needed (yet)